src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Figure implements ColorSupplier, FigureSupplier {

```

Comment:
Figure it's not a subclass of ColorSupplier, FigureSupplier and Figure doesn't implement method of these classes.